addons/mail/push-to-talk-extension/content.js

```diff
@@ -0,0 +1,9 @@
+/* global chrome */
+
+(async () => {
+    window.addEventListener("message", function ({ data }) {
+        if (data.from === "discuss") {
+            chrome.runtime.sendMessage(data);
+        }
+    });
+})();
```

Comment:
What's the purpose of the asynchronous IIFE? If you want to run this after all the synchronous code has been executed, I think you should rather use `setTimeout(..., 0)`. At least the current code looks identical to me as if you called it directly from top-level

Reply:
You're right, leftover from some experimentation. Removed!